src/compiler/types.ts

```diff
@@ -405,6 +405,7 @@ namespace ts {
         JavaScriptFile =     1 << 27,  // If node was parsed in a JavaScript
         ThisNodeOrAnySubNodesHasError = 1 << 28,  // If this node or any of its children had an error
         HasAggregatedChildData = 1 << 29,  // If we've computed data from children and cached it in this node
+        HasJsxSpreadAttribute = 1 << 30,

```

Comment:
Can you log an issue to address the fact that we're now at the effective flag cap for this enum and should break it apart somehow?